### PR TITLE
Fixing the plural/singular form for reading time minutes

### PR DESCRIPTION
**Pull request by ahmgeek**

### CODE REVIEW

**Comment by mkhatib** [comment]
@RashaHussein mind taking a look at this?

**Comment by RashaHussein** [comment]
I agree with you on making them special cases and omitting the number, but I think it's better to say:
"دقيقتان للقراءة" and "دقيقة للقراءة".

Take a look at this comment:
http://goo.gl/QaoZFn

**Comment by AhmedGaber** [comment]
And @ahmgeek, don't forget to [clean your history](https://github.com/manshar/manshar/wiki/Git-Workflow#clean-up-your-history) before submitting the new commit :bowtie: @amrnt 's instructions :joy: :joy:

**Comment by ahmgeek** [comment]
should it be "22 دقيقة قراءة" or its OK to be "22 دقيقة للقراءة" ?

**Comment by RashaHussein** [comment]
I would prefer "22 دقيقة للقراءة"

File: web-client/app/scripts/filters/timeToRead.js [hunk 1]
```diff
@@ -3,6 +3,16 @@
 angular.module('webClientApp')
   .filter('timeToRead', [function () {
     return function(text) {
-      return Math.max(String(text).split(' ').length/200, 1);
+      var time =  Math.max(String(text).split(' ').length/200, 1).toFixed(0);
+      switch(true) {
```
Comment (mkhatib) [inline web-client/app/scripts/filters/timeToRead.js]: I've never seen this usage of the switch statement and it looks somewhat alien. I'd instead rewrote this using if-else statements. Also I would cast the time to integer number after calling toFixed (which will convert it to string).

Comment (ahmgeek) [inline web-client/app/scripts/filters/timeToRead.js]: sorry for the inconvenience :worried: I thought switch will be more suitable for the multiple conditions. 
what about now ?

File: web-client/app/scripts/filters/timeToRead.js [hunk 2]
```diff
@@ -3,6 +3,15 @@
 angular.module('webClientApp')
   .filter('timeToRead', [function () {
     return function(text) {
-      return Math.max(String(text).split(' ').length/200, 1);
+      var time =  +(Math.max(String(text).split(' ').length/200, 1).toFixed(0));
```
Comment (mkhatib) [inline web-client/app/scripts/filters/timeToRead.js]: :+1:

**Comment by mkhatib** [comment]
Awesome work @ahmgeek :space_invader:

**Comment by ahmgeek** [comment]
:see_no_evil: :dancer:

**Pull request merged**